Service/src/main/java/org/example/SetClassId.java

```diff
@@ -1,6 +1,6 @@
 package org.example;
 
 public interface SetClassId {
-
-    
+    void SetId(int id);
```

Comment:
In our hometask first question was: "Any class in your application should be able to have a numeric "ID" field and be able to get and set it. Think about a proper solution for that - abstract class or interface". I have chosen interface because the installation of ID is a common functionality for all classes. But i think that abstract class will be better for this task because in abstract class we could write realization of setting and getting ID for all inherited classes. I think it's worth fixing.

Reply:
Also this sentence doesn't mean that we actually need to extend from this class. We could, but6 if we're not using it what is the reason of adding?)